src/node_sea.cc

```diff
@@ -103,7 +108,10 @@ SeaResource SeaDeserializer::Read() {
   Debug("Read SEA flags %x\n", static_cast<uint32_t>(flags));
   CHECK_EQ(read_total, SeaResource::kHeaderSize);
 
-  std::string_view code = ReadStringView(StringLogMode::kAddressAndContent);
+  std::string_view code =
+      ReadStringView(static_cast<bool>(flags & SeaFlags::kUseSnapshot)
+                         ? StringLogMode::kAddressOnly
+                         : StringLogMode::kAddressAndContent);
   Debug("Read SEA resource code %p, size=%zu\n", code.data(), code.size());
```

Comment:
nit: Same for this one, `resource code` -> `snapshot` when `useSnapshot` is `true`?